src/hotspot/share/cds/filemap.cpp

```diff
@@ -1553,10 +1553,11 @@ void FileMapInfo::write_region(int region, char* base, size_t size,
   int crc = ClassLoader::crc32(0, base, (jint)size);
   if (size > 0) {
     log_info(cds)("Shared file region (%-3s)  %d: " SIZE_FORMAT_W(8)
-                   " bytes, addr " INTPTR_FORMAT " file offset " SIZE_FORMAT_HEX_W(08)
+                   " bytes, addr " INTPTR_FORMAT " file offset 0x%08" PRIxPTR
```

Comment:
If not using one of our FORMAT macros for this one-off place, I think
"0x%08zx" is shorter and at least as understandable as using of one of the PRI
macros (whose meanings I can never remember, and whose very existence I tend
to forget, but maybe that's just me).

But really, I wonder what the point of the field width is here?  Why isn't
this just SIZE_FORMAT_X?

Alternatively, maybe SIZE_FORMAT_X_W should be SIZE_FORMAT_X_0_W?

Reply:
I'm going to take the safer route and not poke around to much in this code. I've inlined the code verbatim, and I'll leave these questions for @iklam, in case he wants to clean this up in the feature.